examples/native-authentication-gmail/server.py

```diff
@@ -66,6 +66,8 @@
 # Use Flask-Dance to automatically set up the OAuth endpoints for Google.
 # For more information, check out the documentation: http://flask-dance.rtfd.org
 google_bp = make_google_blueprint(
+    client_id=app.config["GOOGLE_OAUTH_CLIENT_ID"],
+    client_secret=app.config["GOOGLE_OAUTH_CLIENT_SECRET"],
```

Comment:
If the values were not set on config.json (not wrong credential) would this throw error to end user?

Reply:
@jaykim97 Probably, but we tell the user to provide it via the README: https://github.com/nylas/nylas-python/tree/main/examples/native-authentication-gmail#readme

and if it isn't passed in the call will fail anyways